### DISCUSSION TITLE
production, preview, and development integrations for vercel

### DISCUSSION THREAD

**Discussion by verbiiz** [supabase/supabase#30561] [Feature Requests]
Hey! I'm currently using Vercel + Supabase (great integration btw, fantastic job), and I'm trying to create multiple sets of environment variables within vercel so that I can have separate databases for my Production system and Preview/Development.

however, when I try to link my `my-supa-project` and my dev version `my-supa-project-dev` at the same time, it says only one is allowed (so i would have to manually type all of the environment variables for my new dev project).

pls, could you build support for synchronized env vars with multiple supabase projects? <3

![image](https://github.com/user-attachments/assets/18391b0c-bd2f-47bb-91e2-35235134cdd9)

**Comment by dmaass01** [comment]
Running into the same issues currently. Did you find a better solution in the meantime or is the only way still to type everything manually?
